hammers/osrest/blazar.py

```diff
@@ -68,13 +68,24 @@ def lease_delete(auth, lease_id):
     return response
 
 
+def host_allocations(auth):
+    """Retrieve host allocations"""
+    response = requests.get(
+        url=auth.endpoint('reservation') + '/os-hosts/allocations',
+        headers=dict(X_Auth_Token=auth.token)
```

Comment:
fwiw http headers are case insensitive so you could do `dict(x_auth_token=auth.token)` if you wanted to keep everything lowercase.

Reply:
I used dict() here because it was throwing an error in the python requests package when I used brackets. It was a type error saying headers was getting passed a set. Very weird. Never seen that before. Will use lower case.